h/accounts/views.py

```diff
@@ -315,14 +315,14 @@ def _reset_password(self, user, password):
 class RegisterController(object):
 
     def __init__(self, request):
-        tos_link = ('<a href="/terms-of-service">' +
+        tos_link = ('<a class="link" href="/terms-of-service">' +
                     _('Terms of Service') +
                     '</a>')
-        cg_link = ('<a href="/community-guidelines">' +
+        cg_link = ('<a class="link" href="/community-guidelines">' +
                    _('Community Guidelines') +
                    '</a>')
         form_footer = _(
-            'You are agreeing to be bound by our {tos_link} and '
+            'You are agreeing to our {tos_link} and '
```

Comment:
The length of this footer message causes the "Sign up" button to wrap onto two lines on small screens. We probably want to avoid that.